tools/annotation_tools/template_tool/backend/routes/readAndSaveToFile.js

```diff
@@ -87,4 +98,19 @@ router.post("/writeLabels", function (req, res, next) {
   res.send("post is working properly");
 });
 
+/**
+ * Write labelled pairs
+ */
+router.post("/uploadDataToS3", function (req, res, next) {
+  console.log(req.body);
+  const execSync = require('child_process').execSync;
+  const postprocessing_output = execSync('python ../../../data_processing/autocomplete_postprocess.py', { encoding: 'utf-8' });
```

Comment:
I think the reason the trees were getting printed with unicode on terminal was because of the encoding here being utf-8